de.dlr.sc.virsat.server.feature/feature.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<feature
+      id="de.dlr.sc.virsat.server.feature"
+      label="VirSat Core Headless Rest Jetty Server feature"
+      version="4.10.0.qualifier"
+      provider-name="DLR (German Aerospace Center)"
+      plugin="de.dlr.sc.virsat.branding.ui"
+      license-feature="de.dlr.sc.virsat.license.feature">
+
+   <description>
+      VirSat Jetty Headless Server with REST API using Jersey
+German Aerospace Center (DLR e.V.)
+Simulation and Software Technology
+   </description>
+
+   <copyright>
+      Copyright 2015
```

Comment:
Shouldn't this be a current date? (If it is on purpose, then leave it - just wondering.)

Reply:
Same as above.